src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -501,6 +521,24 @@
   - name: --subnet-name -s
     type: string
     short-summary: The subnet name for the virtual node to use.
+  - name: --appgw-name
+    type: string
+    short-summary: Name of the application gateway to create/use in the node resource group
+  - name: --subnet-prefix
+    type: string
+    short-summary: Subnet Prefix to use for a new subnet created to deploy the Application Gateway
+  - name: --appgw-id
+    type: string
+    short-summary: Resource Id of an existing Application Gateway to use with AGIC
+  - name: --subnet-id
+    type: string
+    short-summary: Resource Id of an existing Subnet used to deploy the Application Gateway
+  - name: --appgw-shared
+    type: string
+    short-summary: This boolean flag should be defaulted to false. Set to true should you need a Shared App Gateway.
```

Comment:
bool? just with --appgw-shared to enable flag, without specifying "true" or "false".